mu4/appshell/qml/HomePage/AddonsContent.qml

```diff
@@ -105,15 +106,16 @@ Rectangle {
                 }
             }
         }
-
-        Rectangle {
+        
+        LanguagesContent {
```

Comment:
BTW I'd prefer to call in LaguagesPage, LanguagesTab or something similar

Reply:
fixed